hedera-node/src/main/java/com/hedera/services/context/MutableStateChildren.java

```diff
@@ -213,4 +213,9 @@ public void updatePrimitiveChildrenFrom(final ServicesState state) {
 		aliases = new WeakReference<>(state.aliases());
 		stakingInfo = new WeakReference<>(state.stakingInfo());
 	}
+
+	/* --- used only in unit tests ---*/
+	public void setNetworkCtx(final MerkleNetworkContext networkCtx) {
```

Comment:
We can annotate with `@VisibleForTesting` to make this more visible and help the IDE find all such methods.

Reply:
Fixed !